mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -52,6 +53,61 @@ def prison_set(user):
     )
 
 
+class JobInformationTestCase(APITestCase, AuthTestCaseMixin):
+    fixtures = [
+        'initial_groups.json',
+        'initial_types.json',
+        'test_prisons.json',
+    ]
+
+    url = reverse('job-information-list')
+
+    payload = {
+        'title': 'Warden',
+        'prison_estate': 'Private',
+        'tasks': 'Run the show',
+    }
+
+    def setUp(self):
+        super().setUp()
+        test_users = make_test_users()
+        self.security_staff = test_users['security_staff'][0]
+        self.security_staff.current_password = 'foobar'
+
+    def test_authorization_required_to_create_job_information(self):
+        response = self.client.post(self.url, data=self.payload)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_can_create_a_job_information_entry(self):
+        response = self.client.post(
+            self.url,
+            data=self.payload,
+            HTTP_AUTHORIZATION=self.get_http_authorization_for_user(self.security_staff)
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(response.data, self.payload)
+
+    def test_will_not_create_with_missing_params(self):
+        self.payload['prison_estate'] = ''
```

Comment:
you're modifying a class-level attribute so all tests that run after this moment are affected! it's safer to just have a different dictionary defined here.

Reply:
Good shout, have updated that to use a fresh dictionary.